blog-collector/2022-10-26

```diff
@@ -0,0 +1,11 @@
+---
+title: Version 19.415-4
+hide_table_of_contents: false
+image: /img/sumo-square.png
```

Comment:
For some reason the relative img link isn't showing up in RSS feeds. I'd like to try the full img link instead.

```suggestion
image: https://help.sumologic.com/img/sumo-square.png
```